5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { mockTodo } from "../../../test/utils";
+import { TodoListItemDriver } from "./TodoListItem.driver";
+
+describe("todo list item component", () => {
+  let driver : TodoListItemDriver;
+
+  beforeEach(() => {
+    driver = new TodoListItemDriver();
+  });
+
+  test("should display description text", async () => {
+    const item = { text: "test" };
+    driver.given.item(item);
+
+    driver.when.render();
+
+    expect(driver.get.todoText()).toEqual(item.text);
+  });
+
+
+  test("should display switchMode status", async () => {
+    driver.when.render();
+
+    expect(driver.get.todoStatus()).toEqual(false);
+  });
+
+
+  test("should display editMode status", async () => {
+    driver.when.render();
+    
+    expect(driver.get.editModeStatus()).toBeFalsy();
+  });
+
+
+  test("should change editMode status to true", async () => {
+    driver.when.render(); 
+    driver.when.editButtonClick();
+
+    expect(driver.get.editModeStatus()).toBeTruthy();
+  });
+
+
+  test("should remove todo item from screen", () => {
```

Comment:
try to describe the test from a product perspective (for instance: should remove an item when clicking the `trash` button)

Reply:
Alright, done that, and for future tests I will come up with better descriptions